### ISSUE TITLE
Fails to find RSS if `www` is missing

### ISSUE THREAD

**Issue by bebraw** [nikezono/node-find-rss#5]
Demo:

```js
'use strict';
var finder = require('find-rss');
var parse = require('parse-rss');


// note, fails without www
finder('http://nixtu.info', function(err, res) {
    if(err) {
        return console.error(err);
    }

    if(res.length) {
        parse(res[0].url, function(err, res) {
            if(err) {
                return console.error(err);
            }

            // title, description, date, categories
            console.log(res.slice(0, 2));
        });
    }
});
```

Note that it works if `www` is provided. Maybe it would make sense to check for both cases?

**Comment by shokai** [comment]
probably, this is fixed latest version.

**Comment by bebraw** [comment]
Cool. Thanks.

**Comment by nikezono** [comment]
I'm sorry for late.
it's solved at `1.5.12`
